patroni/postgresql/config.py

```diff
@@ -336,8 +336,9 @@ def check_directories(self):
         if "stats_temp_directory" in self._server_parameters:
             self.try_to_create_dir(self._server_parameters["stats_temp_directory"],
                                    "'{}' is defined in stats_temp_directory, {}")
-        self.try_to_create_dir(os.path.dirname(self._pgpass),
-                               "'{}' is defined in `postgresql.pgpass`, {}")
+        if "krbsrvname" not in self._config:
```

Comment:
I think it would be more appropriate to use `self._krbsrvname`:
```suggestion
        if not self._krbsrvname:
```